cdap-docs/reference-manual/source/http-restful-api/artifact.rst

```diff
@@ -404,14 +404,13 @@ an HTTP GET request::
      - Optional scope filter. If not specified, defaults to 'user'.
   
 This will return a JSON array that lists the extensions (plugin types) available to the artifact.
-Example output for version |release| of the ``cdap-etl-batch``
-artifact (pretty-printed and reformatted to fit):
+Example output for version |literal-release| of the ``cdap-etl-batch`` artifact:
 
 .. container:: highlight
 
   .. parsed-literal::
-    |$| GET <base-url>/namespaces/default/artifact/WordCount/versions/|release|/extensions?scope=system
-    [ "transform", "validator", "batchsource", "batchsink" ]{
+    |$| GET <base-url>/namespaces/default/artifacts/cdap-etl-batch/versions/|release|/extensions?scope=system
```

Comment:
This is an rst replacement: `|release|` (note the enclosing pipe symbols).

It is replaced with the complete version, including `-SNAPSHOT`, if it is one. If you look on [this page](http://builds.cask.co/artifact/CDAP-DQB28/shared/build-1/Docs-HTML/3.5.0-SNAPSHOT/en/reference-manual/http-restful-api/artifact.html#list-extensions-plugin-types-available-to-an-artifact), you will see that it says "3.5.0-SNAPSHOT".